earth_enterprise/src/SConstruct

```diff
@@ -232,12 +232,14 @@ if gprof:
 
 # setup -rpath-link and -rpath options
 rpathlink_dirs = [Dir(exportdirs['lib']).abspath]
-rpath_dirs = [Dir(exportdirs['lib']).abspath,
-              "'$$ORIGIN/../qt/lib'",
+# SetUID binaries don't use RPATH's containing '$ORIGIN' (since that would be
+# exploitable).  Use absolute paths to library install locations instead:
+rpath_dirs = ["'$$ORIGIN/../qt/lib'",
```

Comment:
Fixing this also may help with rpm linting, as its biggest issue is actually rpaths....